src/_pytest/hookspec.py

```diff
@@ -1,6 +1,19 @@
 """ hook specifications for pytest plugins, invoked from main.py and builtin plugins.  """
+from typing import List
+from typing import Optional
+from typing import Union
+
+import py
```

Comment:
```suggestion
import py.path
```
I've found this to be better with regard to seeing which parts of `py` are used in a file still.